packages/block-library/src/cover/editor.scss

```diff
@@ -62,6 +62,33 @@
 	.wp-block-cover__placeholder-background-options {
 		width: 100%;
 	}
+
+	.wp-block-cover__image--placeholder-image {
+		position: absolute;
+		top: 0;
```

Comment:
I think we only need either `top`, `right`, `bottom`, `left` or `width` and `height`.

Reply:
Fixed in https://github.com/WordPress/gutenberg/pull/41460/commits/2e6cfc759d1e130e87c2c580309bb2f8c9c50685